File: rectangle.cpp
#include "figure.hpp"

double Perimetr(double length, double width) {
    double per = (length + width) * 2;
    return per;
}

double Square(double length, double width) {
    double sqr = length * width;
    return sqr;
}

double Diagonal(double length, double width) {
    double diag = sqrt(pow(length, 2) + pow(width, 2));
    return diag;
}

File: rectangle.py
def rect(array_sides):
    perimetr = (array_sides[0] + array_sides[1])*2
    square = array_sides[0] * array_sides[1]
    diagonal = (pow(array_sides[0], 2)+ pow(array_sides[1], 2))**0.5
    return round(perimetr, 5), round(square, 5), round(diagonal,5)